File: www/app/models/project.rb
class Project

  include Model

  attribute :id, Integer
  attribute :name, String
  attribute :slug, String
  attribute :description, String

  def to_param
    slug
  end

  def tasks
    @tasks ||= Project::Tasks.new(self)
  end

  def followers
    @followers ||= Followers.new(self)
  end

  # def members
  #   @members ||= Members.new(self)
  # end

end

File: www_static/old/javascripts/multify/project.js
Multify.Project = Backbone.Model.extend({
  name: null,
  slug: null,

  initialize: function() {
    this.tasks = new Multify.Tasks;
    this.tasks.project = this;
  }
});

Multify.Project.modelName = 'project';
Multify.Project.path = 'projects';
